src/lib/components/SwitchButton.tsx

```diff
@@ -9,18 +9,13 @@ interface Props {
 export default function SwitchButton({isActive, onClick, size}: Props) {
   return (
     <button
-      className={`relative inline-block ${size === 'sm' ? 'h-2 w-4' : 'h-[24px] w-[45px]'} rounded-full
-    border border-[#e0dce5] bg-white
-    p-0
-    shadow-[inset_0px_1px_2px_rgba(43,34,51,0.04)]
-    transition-[border,box-shadow] duration-100
-    focus:border-[#6c5fc7] focus:shadow-[0px_0px_0px_1px_rgba(108,95,199)]`}
+      className={`relative inline-block ${size === 'sm' ? 'h-2 w-4' : 'h-[24px] w-[45px]'} rounded-full border border-[#e0dce5] bg-white p-0 transition-[border,box-shadow] duration-100 focus:border-[#6c5fc7] `}
```

Comment:
can we use the theme a bit more instead of having raw colors and px values in here?